app/jmeter/jira.jmx

```diff
@@ -6848,14 +6878,22 @@ vars.put(&quot;extend_action&quot;, extend_action);
               <boolProp name="TransactionController.parent">true</boolProp>
             </TransactionController>
             <hashTree>
-              <RandomVariableConfig guiclass="TestBeanGUI" testclass="RandomVariableConfig" testname="random page number (1 - 100)" enabled="true">
-                <stringProp name="variableName">page_number</stringProp>
-                <stringProp name="outputFormat"></stringProp>
-                <stringProp name="minimumValue">1</stringProp>
-                <stringProp name="maximumValue">100</stringProp>
-                <stringProp name="randomSeed"></stringProp>
-                <boolProp name="perThread">false</boolProp>
-              </RandomVariableConfig>
+              <BeanShellPreProcessor guiclass="TestBeanGUI" testclass="BeanShellPreProcessor" testname="get project pages count" enabled="true">
```

Comment:
If I correctly understand we will do this calculation before every browse project request, but result actually will be the same. Is it possible to do this one time per JMeter script run? For example, try to move it to the upper level.

Reply:
Yes, U right! I will move it to the upper level